packages/modules/pricing/integration-tests/__tests__/services/pricing-module/price-list.spec.ts

```diff
@@ -488,6 +503,40 @@ moduleIntegrationTestRunner({
               ]),
             })
           )
+
+          expect(eventBusEmitSpy.mock.calls[0][0]).toHaveLength(4)
+          expect(eventBusEmitSpy.mock.calls[0][0][0]).toEqual(
+            composeMessage(PricingEvents.price_list_created, {
```

Comment:
One suggestion on the event names, maybe we can have them `<entity_name>.<action>` instead of using snake case for all of it. Like that, you can know what is the entity name, and what is the action, so it is a bit cleaner.

Reply:
if you look at the code, the event name is already composed like that `<source>.<object>.<action>`, also, I dont think we should tight the object to the event based on some calculation because it happens to be the case here. wdyt?

Reply:
@adrien2p maybe I am missing something, but if I look in `event-builder-factory.ts` we have
```
isMainEntity
          ? eventsEnum[action]
          : eventsEnum[`${object}_${action}`],
```

I am suggesting we do
```
isMainEntity
          ? eventsEnum[action]
          : eventsEnum[`${object}.${action}`],
```
instead.

Reply:
that would feel weird in my opinion, now if you use the enum to grab an event from the user perspective you would always have to go through `eventEnums['something.created']` instead of being able to do `eventEnums.some_entity_created`. Remember, it is an enum that can be used for subscribers for example

Reply:
It is just a common pattern in naming events in most of the event-driven architecture resources I've read, but I see your point as well. @thetutlage any opinions?

Reply:
@sradevski are you sure you are not mixing up the event name and the enum property name?

Reply:
@sradevski The events are named with dot `.` separator. Its just the enum helpers has `_` convention. Let me share an example:

- Let's say the event is called `product.created`
- Now you want to reference this event somewhere in your code. You have two options.

Either you can write the name manually as

```ts
composeEvent('product.created')
```

Or you may use some Object/enum as a auto-complete unit.

```ts
composeEvent(EventsListEnum['product.created'])
```

Now the above syntax is less ergonomic in typing. Because you have first type square brackets and then maybe get autocomplete for available events list.

Whereas, when this helper use `snake_case`, then it becomes a bit easier to reference. Like this

```ts
composeEvent(EventsListEnum.product_created)
```

Reply:
Yes I was 🤦  😄